Assignment2/juanjoLdlF-JuanJoseLucasdelaFuente.ttl

```diff
@@ -1,21 +1,21 @@
-@base <http://www.ontogrid.net/Ejercicio2#>
-@prefix ejer2: <http://www.ontogrid.net/Ejercicio2#>
-@prefix xsd: <http://www.w3.org/2001/XMLSchema#> 
+@base <http://www.ontogrid.net/Ejercicio2#> .
```

Comment:
No, this final "." is not necessary. I suggest that you use an RDF validation service before submitting again, please